persistence/remote/src/main/java/org/infinispan/persistence/remote/ExternalizerIds.java

```diff
@@ -10,6 +10,6 @@
 
    Integer MIGRATION_TASK = 1900;
    Integer REMOVED_FILTER = 1901;
-   Integer DISCONNECT_REMOTE_STORE = 1902;
-
+   Integer ENTRY_WRITER = 1902;
+   Integer DISCONNECT_REMOTE_STORE = 1903;
```

Comment:
why increasing the DISCONNECT_REMOTE_STORE id? couldn't ENTRY_WRITER be 1093?

Reply:
No reason, it was a merge issue, let me put it back

Reply:
fixed